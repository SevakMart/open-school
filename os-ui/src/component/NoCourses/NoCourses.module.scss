@import "../../styles/Mixins/ButtonMixin.scss";
@import "../../styles/Mixins/FlexMixins.scss";
@import "../../styles/Mixins/FontMixin.scss";

.mainContainer{
    @include flexStyle(column,center, space-between);
    p{
        @include fontStyle('Montserrat',normal,500,1rem,1.5rem);
        color:#5E617B;
    }
    button {
        @include buttonStyle(transparent,#5E617B,1rem,14.125rem,4rem,2rem,2rem,1.375rem,1.375rem);
        @include fontStyle('Montserrat',normal,500,1rem,1.25rem);
        border: 1px solid #5E617B;
        color: #5E617B;
    }
}
