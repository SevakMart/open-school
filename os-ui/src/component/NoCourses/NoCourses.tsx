import NoCoursesIcon from '../../assets/svg/NoCoursesYet.svg';
import { NO_COURSES_YET } from '../../constants/Strings';
import styles from './NoCourses.module.scss';

const NoCourses = () => {
  const { mainContainer } = styles;
  return (
    <div className={mainContainer}>
      <img src={NoCoursesIcon} alt="No Courses Yet" />
      <p>{NO_COURSES_YET}</p>
    </div>
  );
};
export default NoCourses;
