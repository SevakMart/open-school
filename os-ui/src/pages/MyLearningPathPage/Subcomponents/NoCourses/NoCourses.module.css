.mainContainer {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-between;
  margin: 124px 0 0;
}
.mainContainer p {
  font-family: "Montserrat";
  font-style: normal;
  font-weight: 500;
  font-size: 1rem;
  line-height: 1.5rem;
  color: #5E617B;
}/*# sourceMappingURL=NoCourses.module.css.map */