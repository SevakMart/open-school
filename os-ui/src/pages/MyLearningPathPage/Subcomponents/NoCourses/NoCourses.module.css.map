{"version":3,"sources":["NoCourses.module.scss","../../../../styles/Mixins/FlexMixins.scss","NoCourses.module.css","../../../../styles/Mixins/FontMixin.scss","../../../../styles/Configs/Fonts.scss","../../../../styles/Configs/Colors.scss"],"names":[],"mappings":"AAKA;ECJE,aAAA;EACA,sBDIqB;ECHrB,mBDG4B;ECF5B,8BDEoC;EAClC,iBAAA;AEDJ;AFEI;EGPF,yBCDY;EDEZ,kBHOuC;EGNvC,gBHM8C;EGL9C,eHKkD;EGJlD,mBHIuD;EACjD,cKRU;AHYlB","file":"NoCourses.module.css"}