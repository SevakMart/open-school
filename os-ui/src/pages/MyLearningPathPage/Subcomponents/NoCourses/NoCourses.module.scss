@import "../../../../styles/Configs/Colors.scss";
@import "../../../../styles/Configs/Fonts.scss";
@import "../../../../styles/Mixins/FlexMixins.scss";
@import "../../../../styles/Mixins/FontMixin.scss";

.mainContainer{
    @include flexStyle(column,center, space-between);
    p{
        @include fontStyle($primary-font,normal,500,1rem,1.5rem);
        color:$main-text-color;
    }
}
