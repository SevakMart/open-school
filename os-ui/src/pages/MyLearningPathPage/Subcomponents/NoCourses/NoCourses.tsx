import { useTranslation } from 'react-i18next';
import NoCoursesIcon from '../../../../assets/svg/NoCoursesYet.svg';
import styles from './NoCourses.module.scss';

const NoCourses = () => {
  const { t } = useTranslation();
  const { mainContainer } = styles;
  return (
    <div className={mainContainer}>
      <img src={NoCoursesIcon} alt="No Courses Yet" />
      <p data-testid="No courses yet">{t('string.myLearningPaths.noCourseStatus')}</p>
    </div>
  );
};
export default NoCourses;
