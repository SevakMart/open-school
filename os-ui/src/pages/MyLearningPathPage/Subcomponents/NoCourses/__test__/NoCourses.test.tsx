import { render } from '@testing-library/react';
import NoCourses from '../NoCourses';

test('Create a snapshot test', () => {
  const { asFragment } = render(<NoCourses />);
  expect(asFragment()).toMatchSnapshot();
});
