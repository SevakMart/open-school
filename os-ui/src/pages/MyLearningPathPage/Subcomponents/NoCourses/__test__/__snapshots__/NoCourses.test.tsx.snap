// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create a snapshot test 1`] = `
<DocumentFragment>
  <div
    class="mainContainer"
  >
    <img
      alt="No Courses Yet"
      src="NoCoursesYet.svg"
    />
    <p
      data-testid="No courses yet"
    >
      string.myLearningPaths.noCourseStatus
    </p>
  </div>
</DocumentFragment>
`;
